lib/screens/home/mainhome_page.dart

```diff
@@ -130,7 +134,7 @@ Widget _tab(String text, {bool isAllow = false}) {
                   maxRadius: 28.0,
                   foregroundImage: const CachedNetworkImageProvider(""),
                   child: Text(
-                    getInitials(employeeName, 2),
+                    getInitials(employeeName, 2).toUpperCase(),
```

Comment:
@sriLatha2328 toUpperCase function to be moved inside the getInitials function so that it handles the upper cases wherever used.